src/main/java/com/ft/methodearticlemapper/messaging/MessageBuilder.java

```diff
@@ -82,17 +85,22 @@ private Message buildMessage(String uuid, String publishReference, MessageBody m
         public final String contentUri;
         @JsonProperty("lastModified")
         public final String lastModified;
+        @JsonProperty("contentTypeHeader")
```

Comment:
The message content type needs to be set in the `Message` `contentType` field, not in the `MessageBody`